Sieve_ of_Eratosthenes.py

```diff
@@ -0,0 +1,20 @@
+import sys
+sys.stdin = open('input.txt', 'r')
+sys.stdout = open('output.txt', 'w')

```

Comment:
It would be good if you can **also upload** the files necessary to run the script!